catalog/pom.xml

```diff
@@ -49,7 +49,7 @@
         <jaxb.visitor.plugin.version>1.13</jaxb.visitor.plugin.version>
         <catalog.joda.time.version>2.2</catalog.joda.time.version>
         <lucene.bundle.version>5.1.0_1</lucene.bundle.version>
```

Comment:
couldn't find a reference to "lucene.bundle.version" anywhere in ddf, remove the line